src/app/presentation/components/home-components/about/about.component.html

```diff
@@ -46,9 +46,9 @@ <h3 class="feature-title">About team</h3>
             <mat-grid-tile>
               <div class="about-text text-right justify-start">
                 <h3 class="feature-title">About our product</h3>
-                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
-                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore.</p>
-                <p>Lorem ipsum dolor sit amet, consectetur adipiscing elit, sed do eiusmod tempor incididunt ut labore et dolore magna aliqua. Ut enim ad minim veniam, quis nostrud exercitation ullamco laboris nisi ut aliquip ex ea commodo consequat.</p>
+                <p>In my country there is a problem with small buisnes, because of problems with managment. Company owners doesnt want to spending money for managers, justifying this by the fact that their company doesnt so big, and all employers can manage their job on their own. And as a rule, thats strategy leads to the chaos, and total misunderstanding between owners and employees.</p>
```

Comment:
In my country there is a problem with small **_business_**, **_caused by_** problems with managment. Company owners don't want to spend money for managers, explaining that their company isn't so big, and all employees can manage their job on their own. Obviously, this strategy leads to chaos and total misunderstanding among owners and employees.